ShowdownAI.Middleware/ShowdownAI.Middleware/Controllers/ShowdownController.cs

```diff
@@ -9,13 +10,23 @@ namespace ShowdownAI.Middleware.Controllers
     public class ShowdownController() : Controller
     {
         private static readonly MoveSelector _moveSelector = new MoveSelector();
+        private static readonly MoveDataLookup _moveDataLookup = new MoveDataLookup();
 
         [HttpGet]
+        [Route("test")]
         public int Test()
         {
             return 1;
         }
 
+        [HttpGet]
+        [Route("moveDataLookup/{id}")]
+        public MoveData GetMoveInfoById(string id)
+        {
+            return _moveDataLookup.GetMoveData(id);
+        }
+
+
```

Comment:
Could you instead set up a unit test for this? 
So we call the GetMoveData function in the test for a specific value and we know the value that we should get in the future. Should do the same thing but automated then

Reply:
Setup a unit test project and added some initial tests. Will continue to add some more in the future.